modules/ts/misc/run.py

```diff
@@ -850,12 +850,49 @@ def getRunArgs(args):
             path = npath
     return run_args
 
+def moveTests(instance, destination):
+    src = os.path.dirname(instance.tests_dir)
+    # new binaries path
+    newBinPath = os.path.join(destination, "bin")
+
+    try:
+        # copy binaries and CMakeCache.txt to the specified destination
+        shutil.copytree(src, newBinPath)
```

Comment:
Ok, I'll update code to use this functionality for Windows only now.
Also, I'll have a look what can be done for non-Windows systems there in the next contributions.
Could you please provide me with more details (some links, if possible) on the RPATH handling problem you mentioned? And is there some detailed description of how full resulting OCV structure looks like on Linux systems?